target/linux/pistachio/patches-4.4/0000-2-revert-generic-mtd-ubi-add-EOF-marker-support.patch

```diff
@@ -0,0 +1,53 @@
+reverted:
```

Comment:
This is the wrong way to deal with this issue. Even when you revert the patch, the EOF marker is still written to flash as part of the UBI image. If you don't write the EOF marker, the kernel patch is completely harmless.
This really should be fixed in the image building code. 
In LEDE this has been cleaned up properly and EOF markers are only added for devices that actually need them. Please backport those changes in to OpenWrt instead of adding patches like these